.gitignore

```diff
@@ -2,3 +2,4 @@ dist
 node_modules
 storybook-static
 .idea
+.vscode
```

Comment:
I don't think this belongs in the gitignore. That folder allows for config for the repo, and it's a good practice to have it checked in. 

What setting are you using?

Reply:
Ok, I can include it then.